MdeModulePkg/Include/UniversalPayload/SecureBootInfoGuid.h

```diff
@@ -0,0 +1,37 @@
+/** @file
+  This file defines the hob structure for the Secure boot information.
+
+  Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef SECUREBOOT_INFO_GUID_H_
+#define SECUREBOOT_INFO_GUID_H_
+
+#include <UniversalPayload/UniversalPayload.h>
+
+/**
+  Secure Boot info Hob GUID
+**/
+extern EFI_GUID  gUniversalPayloadSecureBootInfoGuid;
+
+#define PAYLOAD_SECUREBOOT_INFO_HOB_REVISION  0x1

```

Comment:
Revision, please follow up other hob naming rule.

Here should be UNIVERSAL_PAYLOAD_SECURE_BOOT_INFO_HOB_REVISION but PAYLOAD_SECUREBOOT_INFO_HOB_REVISION.